### PR TITLE
Some additional validation and UI fixes for Sablier Proposal builder

### PR DESCRIPTION
**Pull request by mudrila**
- Fixes decimal places for stream tranche duration on Sablier proposal builder.
- Disable "Submit" button if there're any invalid streams 
- Minor UI fixes for Sablier proposal builder

Testing these changes:

1. Navigate to "Sablier Proposal Builder", example: http://localhost:8888/proposals/new/sablier/metadata?dao=sep:0x7d0E8e93B5785CB4d5c76f65BB798B3d0a3e89Cb (replace with your DAO address
2. Fill out title and description, proceed to next step
3. Create 1 stream. Fill out token address and amount, input tranches. 
4. Try to put 0 as duration for first tranche - make sure "Create proposal" button disabled
5. Now, put all the valid information, create proposal and make it pass. Verify on Sablier that stream was created with proper tranches durations and amounts

### CODE REVIEW

**Comment by adamgall** [comment]
@mudrila can you please describe how these changes address & fix the issues that Shutter is experiencing? How did you test these changes?

File: src/pages/daos/[daoAddress]/proposals/new/sablier/index.tsx [hunk 1]
```diff
@@ -565,7 +570,9 @@ function StreamBuilder({
                                   isRequired
                                   value={tranche.duration.bigintValue}
                                   placeholder={(SECONDS_IN_DAY * 365).toString()}
-                                  decimalPlaces={1}
+                                  decimalPlaces={0}
```
Comment (mudrila) [inline src/pages/daos/[daoAddress]/proposals/new/sablier/index.tsx:573]: @decentdao/engineering This was causing problems with durations being multiplied by 10
Removing this results in creating stream with proper durations

File: src/pages/daos/[daoAddress]/proposals/new/sablier/index.tsx [hunk 2]
```diff
@@ -851,6 +858,29 @@ export default function SablierProposalCreatePage() {
...
     }
   };
+
+  const invalidStreams = useMemo(
+    () =>
+      streams.filter(stream => {
+        if (!stream.recipientAddress || !stream.tokenAddress || !stream.totalAmount) {
+          return true;
+        }
+        const invalidTranches = stream.tranches.filter((tranche, index) => {
```
Comment (mudrila) [inline src/pages/daos/[daoAddress]/proposals/new/sablier/index.tsx:868]: @decentdao/engineering Potentially - there's so much stuff to cover, but I'm not sure whether it's worth that given this feature is custom implementation specifically for Shutter. Potential things to validate against:
- Checking for treasury balance and disabling the button(or showing warning?) if treasury balance is less than sum of tokens in multiple streams
- Checking tranches amounts
- Showing total duration of the stream
On the other hand - well, once I thought that it's not reasonable - and here we are, with this PR 🤣 
Any thoughts?

**Comment by mudrila** [comment]
> @mudrila can you please describe how these changes address & fix the issues that Shutter is experiencing? How did you test these changes?

@adamgall I've added some comments for the code and testing steps - please review :)

File: src/pages/daos/[daoAddress]/proposals/new/sablier/index.tsx (hunk 2 above)
Comment (adamgall) [reply src/pages/daos/[daoAddress]/proposals/new/sablier/index.tsx:868]: This validation looks like the appropriate level of validation to me.

**Review by adamgall — APPROVED** [review]
Approving this, assuming the code has been manually tested (by you @mudrila), using the inputs that Shutter shared with you that they'll be using on Mainnet.

File: src/pages/daos/[daoAddress]/proposals/new/sablier/index.tsx (hunk 2 above)
Comment (Da-Colon) [reply src/pages/daos/[daoAddress]/proposals/new/sablier/index.tsx:868]: (nitpick) could these filters be `.some`?

**Review by Da-Colon — COMMENTED** [review]
Nicpick but otherwise code looks good to me.

**Review by DarksightKellar — APPROVED** [review]
LGTM

**Pull request merged**